services/src/api/model/responses/ml_models/mod.rs

```diff
@@ -0,0 +1,19 @@
+use serde::{Deserialize, Serialize};
+use utoipa::ToResponse;
+
+use crate::machine_learning::name::MlModelName;
+
+#[derive(Debug, Serialize, Deserialize, Clone, ToResponse)]
```

Comment:
add ToSchema and add to apidoc.rs if necessary

Reply:
https://github.com/geo-engine/geoengine/pull/1007/commits/854afecaa6351ca243c17a8fce0840f5a2282d92

Reply:
https://github.com/geo-engine/geoengine/pull/1007/commits/7955102e5a5049100a712e8fca5735846c6ad5ca